app/controllers/api_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ApiController < ApplicationController
+  def random_greeting
+    render json: { greeting: Message.order('RANDOM()').first.greeting }
```

Comment:
- [ ] Great job working so far but I think you have an error here since greeting is undefined method so kindly solve the problem to make your app work as expected ✨🤩

## This error shown to me ⬇⬇
![image](https://user-images.githubusercontent.com/58553711/222026344-416fb3c9-d3b2-4a84-b302-753244e7cb24.png)